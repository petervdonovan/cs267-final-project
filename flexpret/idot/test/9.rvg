[mu []
[[file {Reset_Handler} {
  Reset_Handler:
    [id init]
    [single t5
      [iterate t1 of [cyclify a0 t6 [rng-seeded t2 t3 bits 5 length 3 seeded 7] times 3]
        {
          [critical-hart t0 t5
            {
              [sim-print t1 t4]
            }]
        }]]
    rdcycle t5
    [critical-hart t6 a1 [sim-print t5 a0]]
    [id die]}]]
]
