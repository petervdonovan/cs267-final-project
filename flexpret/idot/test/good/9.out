Core-0:         19
Core-0:         20
Core-0:         10
Core-0:         19
Core-0:         20
Core-0:         10
Core-0:         19
Core-0:         20
Core-0:         10
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0:        713
Core-1:        713
Core-2:        713
Core-3:        713
Core-4:        713
Core-5:        713
Core-6:        713
Core-7:        713
Core-8:        713
Core-9:        713
Core-10:        713
Core-11:        713
Core-12:        713
Core-13:        713
Core-14:        713
Core-15:        713
Core-0 is done
Core-1 is done
Core-2 is done
Core-3 is done
Core-4 is done
Core-5 is done
Core-6 is done
Core-7 is done
Core-8 is done
Core-9 is done
Core-10 is done
Core-11 is done
Core-12 is done
Core-13 is done
Core-14 is done
Core-15 is done
All cores are done terminating
Assertion failed: Program terminated sucessfully
